products/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = 'categories'
+
+class SubCategory(models.Model):
+    name     = models.CharField(max_length=100)
+    category = models.ForeignKey('Category', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'sub_categories'
+
+class Product(models.Model):
+    name         = models.CharField(max_length=100)
+    description  = models.CharField(max_length=500)
+    content_url  = models.CharField(max_length=200)
+    price        = models.IntegerField()
```

Comment:
```suggestion
    price        = models.DecimalField(...)
```
price는 DecimalField로 사용해주세요.